src/ui/wxWidgets/MenuViewHandlers.cpp

```diff
@@ -226,7 +226,7 @@ void PasswordSafeFrame::RunShowReport(int iAction)
   }
   else {
     wxString tcAction = CReport::ReportNames.find(iAction)->second;
-    wxMessageBox(_(tcAction) + _(L" file \'") + rpt.GetFileName() + _(L"' not readable"), _("View Report"), wxOK|wxICON_ERROR);
+    wxMessageBox(tcAction + _(L" file \'") + rpt.GetFileName() + _(L"' not readable"), _("View Report"), wxOK|wxICON_ERROR);
```

Comment:
Concatenating localizable strings is an anti-pattern. This should use a template string like most other UI captions.